server/utils/validation/index.js

```diff
@@ -1,3 +1,8 @@
+const appointmentsSearchSchema = require('./appointmentsSearchSchema');
+
+module.exports = {
+  appointmentsSearchSchema,
+};
```

Comment:
change the way you are exporting modules with it to the way your teammates using it.